webapp/src/views/Home.vue

```diff
@@ -359,6 +364,18 @@ export default {
         text: "We provide easy integration with Let's Encrypt and their certbot tool.",
       },
     ],
+    breaking_news: [
+      {
+        id: 'news-20200604001',
+        start: new Date(Date.UTC(2020, 6 - 1, 4)),  // first day of showing
```

Comment:
You can just use something like `new Date('2020-06-04')` or `new Date('2020-06-04T02:00:00+02:00')`

Reply:
The usage of that is [strongly discouraged](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) by Mozilla, and I'm not sure how it goes together with UTC dates

Reply:
I immediately believe there are browser incompatibilities, if you don't specify a timezone. However, if you do (UTC is denoted by `Z` after time part of the string), then I can't actually imagine any ambiguities.

It's not discouraged because it's going to disappear, but because the behavior for certain inputs is not well defined. I think it would be fine if we use it with "fully qualified date-times" only. But if you feel uncomfortable with that, let's stick to what you have.